multidict/_multidict.c

```diff
@@ -709,13 +709,21 @@ static inline void
 multidict_tp_dealloc(MultiDictObject *self)
 {
     PyObject_GC_UnTrack(self);
+#if PY_MAJOR_VERSION >= 3 && PY_MINOR_VERSION >= 9
+    Py_TRASHCAN_BEGIN(self, multidict_tp_dealloc)
+#else
     Py_TRASHCAN_SAFE_BEGIN(self);
+#endif
     if (self->weaklist != NULL) {
         PyObject_ClearWeakRefs((PyObject *)self);
     };
     pair_list_dealloc(&self->pairs);
     Py_TYPE(self)->tp_free((PyObject *)self);
+#if PY_MAJOR_VERSION >= 3 && PY_MINOR_VERSION >= 9
+    Py_TRASHCAN_END // there should be no code after this
```

Comment:
And here?
```suggestion
    Py_TRASHCAN_END;  // there should be no code after this
```

Reply:
Ah, nevermind. Found the macro definition and it has a semicolon:
https://github.com/python/cpython/blob/b5dc0f83adda0bde1b176c3a84cc31cd94254e73/Include/cpython/object.h#L474-L478